sdk/keyvault/keyvault-certificates/README.md

```diff
@@ -338,7 +338,7 @@ Let's evaluate the composition of a Key Vault Certificate:
 
 Knowing that the private key is stored in a Key Vault Secret,
 with the public certificate included, we can retrieve it
-by using the [KeyVault Secrets client][keyvault-secrets-client].
+by using the KeyVault Secrets client.
```

Comment:
When referring to the service, we should use "Key Vault" as two separate words, unless it's in something like a class 
 name, package name, etc.

Reply:
Noted, I did a search through all the READMEs and replaced them. There were several occurrences of this.